apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerTypeMapper.cs

```diff
@@ -21,9 +21,17 @@
 namespace Microsoft.EntityFrameworkCore.Storage.Internal
 {
     /// <summary>
+    /// This is internal functionality and not intended for public use.
     /// </summary>
     public class SpannerTypeMapper : RelationalTypeMapper
     {
+        //Note: This SpannerTypeMapper serves a key role in all of queries, updates and even migration by handling the
+        // conversion between ClrTypes and SpannerTypes.  While this class is fairly simple, it relies heavily on
+        // the type conversion support supplied in SpannerDbType which automatically coerces values into their
+        // appropriate form across the wire and when coming back.
+
+        //TODO: Decimal unfortunately loses precision with FLOAT64 and it would be nice to replace with something better.
```

Comment:
That's odd - decimal has more precision than double, but less range. I'd expect that within the expected range of decimal, it would be fine. Will leave the TODO for now but look in the future.

Reply:
What I mean is the spanner has no type support for decimal and representing c# decimal with a FLOAT64 will lose precision and is not ideal.

Reply:
Ah, I see what you mean in that case. Thanks!